src/firefly/java/edu/caltech/ipac/firefly/data/Version.java

```diff
@@ -75,6 +76,9 @@ public void setBuildCommitFirefly(String _buildCommitFirefly) {
     }
 
     public void setBuildFireflyTag(String buildFireflyTag) { this._buildFireflyTag = buildFireflyTag; }
+
+    public void setBuildFireflyBranch(String buildFireflyBranch) { this._buildFireflyBranch = buildFireflyBranch; }
```

Comment:
FYI only.  Don't change.
This is the convention.  Whichever one is fine.  There's no wrong answer.
c++:
```c++
int _a;
function setA(int a) { _a = a; }          // in this case, this._a is not needed.
```
Java:
```c++
int a;
function setA(int a) { this.a = a; }
```